src/App.vue

```diff
@@ -13,3 +13,17 @@ export default defineComponent({
   },
 });
 </script>
+
+<style lang="scss">
+html {
+  height: 100%;
+}
+
+body {
+  height: 100%;
+}
```

Comment:
Not sure if you are using a CSS reset? Maybe it can be tried...

Reply:
I'm not sure but I think that quasar is already using a normalize.sass file

![image](https://user-images.githubusercontent.com/66505715/172068692-bba91f71-a5f4-4900-99b0-ff233978b6e5.png)

---

The code that you're selecting on your comment is to get a full-height screen size, but it's not working since when the page is bigger it looks so:

![image](https://user-images.githubusercontent.com/66505715/172068989-b74a97c4-76f3-4b5d-9797-e74305e0f34c.png)

I will try other solutions, if you've some suggestions I'm open to hearing them.

---

So thanks for taking the time to review !

Reply:
I've made a change to solve the bug when mainLayout is bigger than 100vh

![image](https://user-images.githubusercontent.com/66505715/172069239-81a29ab4-b862-4ecc-97f9-879cf3966228.png)

Is cool that it was working since is a just CSS solution and it's easy to make (easier than min-height: 100vh, since it cause problems on mobile. I need to check if this solution is ok on mobile)